core/lib/sif/sif.c

```diff
@@ -111,7 +111,8 @@ sif_validate(Sifinfo *info)
 		currarch = SIF_ARCH_386;
 	else if(!strncmp(name.machine, "arm", 3) && sizeof(void *) == 4)
 		currarch = SIF_ARCH_ARM;
-	else if(!strncmp(name.machine, "arm", 3) && sizeof(void *) == 8)
+	else if( (!strncmp(name.machine, "arm", 3) && sizeof(void *) == 8) ||
```

Comment:
there's a extra space after if( that doesn't match the rest of the if statement.

I am not sure if we want all machines being converted to AARCH64. It seems like it's a different case.